DATA/production/qc-postproc-async/zdc.json

```diff
@@ -0,0 +1,226 @@
+          {
+            "name": "details",
+            "title": "ZDC DETAILS",
+            "path": "ZDC/QO",
+            "ignoreQualitiesDetails": [],
+            "inputObjects": [
+              {
+                "name": "QcZDCRecCheck",
+                "title": "Rec check"
+              }
+            ]
+          }
+        ],
+        "initTrigger": [
+          "userorcontrol"
+        ],
+        "updateTrigger": [
+          "newobject:qcdb:ZDC/QO/ZDCQuality/ZDCQuality"
+        ],
+        "stopTrigger": [
+          "userorcontrol"
+        ]
+      }
+    },
+      "checks": {
+      "RecCheck": {
+        "active": "true",
+        "className": "o2::quality_control_modules::zdc::ZDCRecDataCheck",
+        "moduleName": "QcZDC",
+        "detectorName": "ZDC",
+        "policy": "OnAny",
+        "checkParameters": {
+          "ADC_ZNAC": "103.6;10;20",
+          "ADC_ZNA1": "129.8;10;20",
+          "ADC_ZNA2": "131.4;10;20",
+          "ADC_ZNA3": "137.8;10;20",
+          "ADC_ZNA4": "128.6;10;20",
+          "ADC_ZNAS": "427.7;10;20",
+          "ADC_ZPAC": "101.8;10;20",
+          "ADC_ZPA1": "166.6;10;20",
+          "ADC_ZPA2": "165.2;10;20",
+          "ADC_ZPA3": "132.7;10;20",
+          "ADC_ZPA4": "135.3;10;20",
+          "ADC_ZPAS": "421.4;10;20",
+          "ADC_ZEM1": "101.1;10;20",
+          "ADC_ZEM2": "96.82;10;20",
+          "ADC_ZNCC": "99.48;10;20",
+          "ADC_ZNC1": "131.1;10;20",
+          "ADC_ZNC2": "127.8;10;20",
+          "ADC_ZNC3": "132.4;10;20",
+          "ADC_ZNC4": "135.4;10;20",
+          "ADC_ZNCS": "414.9;10;20",
+          "ADC_ZPCC": "102.2;10;20",
+          "ADC_ZPC1": "134.8;10;20",
+          "ADC_ZPC2": "135.7;10;20",
+          "ADC_ZPC3": "164.3;10;20",
+          "ADC_ZPC4": "169.0;10;20",
+          "ADC_ZPCS": "424.2;10;20",
+          "ADC_POS_MSG_X": "0.15",
+          "ADC_POS_MSG_Y": "0.92",
+          "TDC_ZNAC": "-0.66;0.10;0.20",
+          "TDC_ZNAS": "-1.36;0.10;0.20",
+          "TDC_ZPAC": "-0.94;0.10;0.20",
+          "TDC_ZPAS": "-0.17;0.10;0.20",
+          "TDC_ZEM1": "-1.08;0.10;0.20",
+          "TDC_ZEM2": "-0.26;0.10;0.20",
+          "TDC_ZNCC": "0.53;0.10;0.20",
+          "TDC_ZNCS": "-0.89;0.10;0.20",
+          "TDC_ZPCC": "1.11;0.10;0.20",
+          "TDC_ZPCS": "0.87;0.10;0.20",
+          "TDC_POS_MSG_X": "0.01",
+          "TDC_POS_MSG_Y": "0.92"
+        },
+        "dataSource": [
+          {
+            "type": "PostProcessing",
+            "name": "RecPP",
+            "MOs": [
+              "h_summary_ADC", "h_summary_TDC"
+            ]
+          }
+        ]
+      }
+    },
+    "aggregators": {
+      "ZDCQuality": {
+        "active": "true",
+        "className": "o2::quality_control_modules::common::WorstOfAllAggregator",
+        "moduleName": "QcCommon",
+        "policy": "OnAll",
+        "detectorName": "ZDC",
+        "dataSource": [
+          {
+            "type": "Check",
+            "name": "QcZDCRecCheck"
```

Comment:
Make sure the name is consistent with the name of the check task. It's a general comment. Please check all the names are consistent.

Reply:
done